# expect: a display list records a run and replays it into another renderer
--- a/ffi/zrecast_internal.h
+++ b/ffi/zrecast_internal.h
@@ -794,7 +794,6 @@
   void vertex(const float x, const float y, const float z,
               unsigned int color) override {
     duDisplayList::vertex(x, y, z, color);
-    ++count_;
   }
 
   void vertex(const float* pos, unsigned int color) override {
